inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -421,16 +418,23 @@ def _interpolate_to_uniform_grid(
         self,
         x_resolution: float,
         y_resolution: float,
+        threshold_distance: Optional[float] = None,
```

Comment:
This whole function seems to be agnostic of the bathymetry, should it also leave in `inductiva.utils.interpolation`?

Reply:
Hm, maybe! It has behaviour specific to the bathymetry (for now), like the fact that it can't (in some situations) be interpolated. We can move it in the future, when similar behaviour is required by some other scenario / utility. WDTY?

Reply:
> Hm, maybe! It has behaviour specific to the bathymetry (for now), like the fact that it can't (in some situations) be interpolated. We can move it in the future, when similar behaviour is required by some other scenario / utility. WDTY?

Those situations are not specific of the bathymetry, are more related with the non-existence of points, no?
That is perfectly fine!